.buildkite/pipeline.build.yml

```diff
@@ -45,7 +45,7 @@
       --test_env=DOCKER_TLS_CERTDIR=/certs
       -- python/ray/tests/...
 
-- label: ":book: Doctest (CPU)"
+- label: ":book: Doctest (no owners)"
```

Comment:
what does no owners mean? has no team tag?

Reply:
like these https://github.com/ray-project/ray/blob/master/python/ray/experimental/BUILD#L5, they have `team:none` as a tag

Reply:
Most of the code snippets under `doc/` technically don't have an owner. The historical reason for this was that we tested all of the `doc/` code snippets with a single `doctest` rule. Since it wasn't clear what to mark the team as, we left it as `none`.

Going forward, we should probably split up `["source/**/*.rst", "source/**/*.md"]` for each teams?

https://github.com/ray-project/ray/blob/79a3c58f5cd0a81422f917fc2b22f7272bb98981/doc/BUILD#L287-L314

Reply:
Yess, splitting up sounds good. Everything should have owners. I will follow up to fix this.